src/components/Application/ApplicationStep.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Container, Grid, Header, Icon, Segment } from 'semantic-ui-react'
+import { ApplicationQuestion, Loading } from '../../components'
+import { TemplateElement, useGetSectionElementsQuery } from '../../utils/generated/graphql'
+import { CurrentSectionPayload } from '../../utils/types'
+
+interface ApplicationStepProps {
+  currentSection: CurrentSectionPayload
+  elements: TemplateElement[]
+  onNextClicked: (() => void) | null
+  onPreviousClicked: (() => void) | null
+}
+
+const ApplicationStep: React.FC<ApplicationStepProps> = (props) => {
+  const { currentSection, elements, onNextClicked, onPreviousClicked } = props
+
+  return (
+    <Container textAlign='left'>
+      <Segment>
+        <Header content={currentSection.title} />
+        {elements.map(element => <ApplicationQuestion templateElement={element}/>)}
+        <Grid columns={3}>
+          <Grid.Row>
+            <Grid.Column>
+              {onPreviousClicked && <PageButton title='Previous' type='left' onClicked={onPreviousClicked}/>}
+            </Grid.Column>
+            <Grid.Column/>{/* Empty cell */}
+            <Grid.Column>
+              {onNextClicked && <PageButton title='Next' type='right' onClicked={onNextClicked}/>}
+            </Grid.Column>
+          </Grid.Row>
+        </Grid>
+      </Segment>
+    </Container>
+  )
+}
+
+type ButtonDirection = 'left' | 'right'
+
+interface PageButtonProps {
+  title: string
+  type: ButtonDirection
+  onClicked: () => void
+}
+
+const PageButton: React.FC<PageButtonProps> = (props) => {
+  const { title, type, onClicked } = props
+  return (
+    <Button 
+      labelPosition={type} 
+      icon ={type==='right'?'right arrow' : 'left arrow'} 
```

Comment:
Have you got your Prettier set up okay? I noticed this file doesn't conform to the usual formatting (e.g. no spacing around the `?` here, and when I saved this file it did quite a bit of re-formatting.

(FWIW -- I suspect the latest VSCode update might have done something weird with default formatting. My wasn't working properly yesterday either, and I had to re-select Prettier as my default formatter to get it going again. 🤷‍♂️)

Reply:
Yes. Thanks for helping with that. Sorted. I'll run the prettier over this one now.